pkg/storaged/dialog.jsx

```diff
@@ -537,8 +493,30 @@ export const ComboBox = (tag, title, options) => {
 
         render: (val, change) =>
             <div data-field={tag} data-field-type="combobox">
-                <ComboboxElement value={val} choices={options.choices}
-                                 disabled={options.disabled} onChange={change} />
+                <TypeAheadSelect
+                    id="nfs-path-on-server"
+                    labelKey="path"
+                    placeholder=""
+                    paginate={false}
+                    onChange={value => change(value[0])}
+                    onInputChange={change}
+                    options={options.choices}
+                    disabled={options.disabled}
+                    onKeyDown={ev => { // Capture ESC event
```

Comment:
So I know I wrote this code for the file autocompletion but I was just testing it and it doesn't seem to work (neither in the file autocompletion component)
I meaning that hitting Esc will just close the dialog, which is not what we want here.

Reply:
Ech yeah, I shamelessly stole it from file autocomplete component :D 
I prefer to fix it together outside of this PR

Reply:
Sure, let's fix it separately, I wouldn't have realized it's broken if it was not for your PR anyway :)